src/Commands/CommandFactory.cs

```diff
@@ -22,7 +22,7 @@ public class CommandFactory
     private readonly CommandGroup _rootGroup;
     private readonly ModelsJsonContext _srcGenWithOptions;
 
-    internal const char Separator = '-';
+    internal const char Separator = '_';
```

Comment:
I believe there's another place in code that looks for the dash - when it converts from the slug to the command name.

Reply:
do you remember the hints? or pattern I could search?

Reply:
i think it is in the servercommand code where it maps tools to commands.

Reply:
do you mean the usage of "StdioClientTransport" https://github.com/search?q=repo%3AAzure%2Fazure-mcp%20StdioClientTransport&type=code ?